src/test/java/com/baidu/hugegraph/api/IndexLabelApiTest.java

```diff
@@ -445,4 +448,40 @@ public void testAddIndexLabelWithUserDataWithApiVersion49() {
 
         client().apiVersion(originApiVersion);
     }
+
+    @Test
+    public void testAddIndexLabelWithRebuildFalse() {
+        Vertex vertex = new Vertex("person");
+        vertex.property("name", "James");
+        vertex.property("city", "Beijing");
+        vertex.property("age", 19);
+
+        vertex = vertexAPI.create(vertex);
+
+        IndexLabel personByAge = schema().indexLabel("personByAge")
+                                         .onV("person")
+                                         .by("city")
+                                         .secondary()
+                                         .rebuild(false)
+                                         .build();
+        IndexLabel.CreatedIndexLabel il = indexLabelAPI.create(personByAge);
+        IndexLabel created = il.indexLabel();
+        Assert.assertEquals(personByAge.name(), created.name());
+        Assert.assertEquals(personByAge.baseType(), created.baseType());
+        Assert.assertEquals(personByAge.baseValue(), created.baseValue());
+        Assert.assertEquals(personByAge.indexType(), created.indexType());
+        Assert.assertEquals(0, il.taskId());
```

Comment:
seems the server returned null

Reply:
taskId is long type. 0 means server return null